packages/cms/form/date-picker/date.tsx

```diff
@@ -0,0 +1,33 @@
+import { EuiDatePicker } from '@tensei/eui/lib/components/date_picker'
+import { EuiFormRow } from '@tensei/eui/lib/components/form'
+import React, { useState } from 'react'
+
+import moment from 'moment'
+import { FormComponentProps } from '@tensei/components'
+
+const FormDatePicker: React.FC<FormComponentProps> = ({
+  field,
+  value,
+  onChange
+}) => {
+  const [startDate, setStartDate] = useState(moment.utc(value))
+
+  const handleChange = (date: moment.Moment) => {
+    setStartDate(date)
+    onChange(date.format())
+  }
+
+  return (
+    <EuiFormRow fullWidth>
+      <EuiDatePicker
```

Comment:
So date pickers are a little more complicated. 

If you see the [docs here for date field](https://tenseijs.com/docs/fields#date-field), you'll notice there's a format, so when calling onChange we need to respect the format from the Tensei app. Also we need to respect the `defaultToNow` configuration. 

There's also some options like `timePicker24Hr()` to show if there's a 24H time picker, also if its a dateTime or timestamp field it'll show a time picker.